docs/contributing.md

```diff
@@ -29,12 +29,19 @@ Please just be sure to:
 * follow the style, naming and structure conventions of the rest of the project.
 * make commits atomic and easy to merge.
 * use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for the PR title. This will help us to understand the nature of the changes, and to generate the changelog after all the commits in the PR are squashed.
+    * Please use the `break` type for breaking changes, as these categories are considered as `breaking change` in the changelog.
```

Comment:
suggestion: document the exclamation mark and BREAKING CHANGE: footer instead of `break` type.

Reply:
Forgot this! Will change it tomorrow, good catch